src/Raven.Server/Documents/TcpHandlers/SubscriptionConnection.cs

```diff
@@ -788,18 +797,33 @@ private void AssertCloseWhenNoDocsLeft()
             switch (clientReply.Type)
             {
                 case SubscriptionConnectionClientMessage.MessageType.Acknowledge:
+                {
+                    if (ClusterCommandsVersionManager.CurrentClusterMinimalVersion >= 53_000)
                     {
                         await _processor.AcknowledgeBatch(CurrentBatchId);
+                    }
+                    else
+                    {
+                        await TcpConnection.DocumentDatabase.SubscriptionStorage.LegacyAcknowledgeBatchProcessed(
+                            SubscriptionId,
+                            Options.SubscriptionName,
+                            LastSentChangeVectorInThisConnection ?? nameof(Client.Constants.Documents.SubscriptionChangeVectorSpecialStates.DoNotChange),
```

Comment:
why dont we use `subscriptionChangeVectorBeforeCurrentBatch` ?

Reply:
@lastav5 can you have a look at this comment?

Reply:
@arekpalinski 
Yes, We've reached an agreement that this isn't needed after all

Reply:
ok